internal/reader/adapter_v1.go

```diff
@@ -40,9 +41,18 @@ func NewAdapterV1(
 	if err != nil {
 		panic(err)
 	}
+	voucherRepository := &cRepos.VoucherRepository{
```

Comment:
It's not your fault, however, the container is responsible for creating repository instances...